index.md

```diff
@@ -32,6 +32,7 @@ Vendor | Base Pricing | SSO Pricing | % Increase | Source | Date Added
 [DocuSign](https://www.docusign.com) | $25 per u/m | $50 per u/m | 100% | [🔗](https://www.docusign.com/products-and-pricing) Quote | 2018-10-17
 [Dropbox](https://www.dropbox.com) | $15 per u/m | $25 per u/m | 67% |  [🔗](https://www.dropbox.com/business/pricing) | 2018-10-17
 [Expensify](https://www.expensify.com) | $5 per u/m | $9 per u/m | 80% | [🔗](https://www.expensify.com/pricing#features) | 2018-10-17
+[Github](https://www.github.com) | $9 per u/m | $21 per u/m | 263% | [🔗](https://github.com/pricing) | 2018-10-19
```

Comment:
```suggestion
[Github](https://www.github.com) | $9 per u/m | $21 per u/m | 133% | [🔗](https://github.com/pricing) | 2018-10-19
```

% increase is "percentage of original price added to original price" at the moment, which I appreciate is not obvious.